contracts/TokenLockingStorage.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.4.23;
+
+import "../lib/dappsys/auth.sol";
+
+
+contract TokenLockingStorage is DSAuth {
+  address resolver;
+
+  // Address of ColonyNetwork contract
+  address colonyNetwork;
+
+  // Maps token to user to amount locked
+  mapping (address => mapping (address => uint256)) depositedBalances;
+
+  // Maps token to user to number of locks
+  mapping (address => mapping (address => uint256)) userTokenLockCount;
```

Comment:
Sorry just noticed that if we merge these two mappings it'll be much cheaper gas-wise. So have a
```
struct TokenLock {
    uint256 amount;
    uint256 counter;
  }
mapping (address => mapping (address => TokenLock)) userTokenLocks;
```

Reply:
Awsome, thanks 👍